sklearn/neighbors/base.py

```diff
@@ -101,13 +101,14 @@ def _get_weights(dist, weights):
 class NeighborsBase(six.with_metaclass(ABCMeta, BaseEstimator)):
     """Base class for nearest neighbors estimators."""
 
-    @abstractmethod
-    def __init__(self):
-        pass
+    _fit_X = None
```

Comment:
On the previous version, these variables are setted into None in `__init__`. I initialised them to prevent unknown behavior.